web/Views/Shared/Details/_TenantPreferences.cshtml

```diff
@@ -35,7 +35,7 @@
         {
             <div class="@GetPreferenceFont(Model.Property.LandlordRequirements.AcceptsUnder35)">
                 <i class="bi me-1 @GetPreferenceIcon(Model.Property.LandlordRequirements.AcceptsUnder35)"></i>
-                Is over 35
+                Is under 35

```

Comment:
What's the reason for this change? Is this one that was missed out before?

Reply:
Yeah this was missed out I think. Its different on the other property pages